literature.bib

```diff
@@ -24923,29 +24934,39 @@ @online{bluej
 	title = {{Bluej IDE}},
 	author = {King's College London},
 	howpublished = {\url{https://www.bluej.org/}},
+	url = {https://www.bluej.org/},
+	date = {2020-01-06},
+	year = 2020,
 	note = {Accessed at January 06, 2020}
 }
 
 @online{marlin,
 	title = {{Marlin Firmware}},
 	author = {Erik van der Zalm},
 	howpublished = {\url{http://marlinfw.org/}},
+	url = {http://marlinfw.org/},
+	date = {2019-12-02},
+	year = 2019,
 	note = {Accessed at December 02, 2019}
 }
 
 @online{linux,
 	title = {{Linux Operating System}},
 	author = {Linus Torvalds},
-	howpublished = {\url{www.kernel.org}},
+	howpublished = {\url{https://www.kernel.org/}},
+	url = {https://www.kernel.org/},
+	date = {2019-12-02},
+	year = 2019,
 	note = {Accessed at December 02, 2019}
 }
 
 @online{apoweb,
 	author = {Dirk Aporius},
-	month = FEB,
 	title = {{Apo-Games}},
 	year = 2020,
 	howpublished = {\url{http://apo-games.de/}},
+	url = {http://apo-games.de/},
+	date = {2020-06-25},
 	note = {Accessed at June 25, 2020}
 }
 
```

Comment:
I did not look into all of the changes here. I guess we have to test it with our papers.